packages/documentation/docs/apis/auth/enums.md

```diff
@@ -0,0 +1,70 @@
+---
+id: enums
+title: Enums
+slug: enums
+sidebar_position: 6
+---
+
+<!-- Do not edit this file, it has been automatically generated by docusaurus-graphql-plugin -->
```

Comment:
I'm surprised to see this file addition and docusaurus comment since we migrated to astro from docusaurus. Was this generated using the pre-astro migration documentation package? Maybe you need to merge the latest in from main?

I guess im not complete sure what's going on because the auth enums.md file still exists here on main: https://github.com/interledger/rafiki/blob/main/packages/documentation/src/content/docs/apis/auth/enums.md But the gql schemas are generated by graphql-markdown and output here: https://github.com/interledger/rafiki/blob/main/packages/documentation/src/schema-dumps/auth_raw_schema.md

I'm surprised we still have the docusaurus generated stuff (as it will not be updated when we change the schema).